rosmon_core/test/xml/test_param.cpp

```diff
@@ -44,8 +44,8 @@ TEST_CASE("param_types", "[param]")
 			<param name="str_param_auto" value="hello" />
 			<param name="str_param_forced" value="0" type="str" />
 
-			<param name="bool_param_auto" value="true" />
-			<param name="bool_param_forced" value="true" type="boolean" />
+			<param name="bool_param_auto" value="True" />
```

Comment:
This way we loose the tests for lowercase true... Can we have separate tests for lowercase/uppercase here?